### PR TITLE
Factory icon instead of folder on actions view

### PR DESCRIPTION
**Pull request by lanwen**
Now it looks:
image

image

[Review on Reviewable](https://reviewable.io/reviews/jenkinsci/job-dsl-plugin/497)

### CODE REVIEW

**Comment by daspilker** [comment]
Where does the icon come from? What about copyright?

**Comment by lanwen** [comment]
http://www.iconspedia.com/icon/factory-icon-37191.html

Where can i write this info? (License says we should only do is write author and license name) 
```
Author: IconsPedia, http://www.iconspedia.com
License: CC Attribution
```

**Comment by daspilker** [comment]
I'm not sure, please ask on the [Jenkins Dev](https://groups.google.com/forum/?fromgroups#!forum/jenkinsci-dev) list.

**Comment by lanwen** [comment]
https://groups.google.com/forum/?fromgroups#!topic/jenkinsci-dev/o59PXj-UpqI

**Comment by daniel-beck** [comment]
This doesn't really fit in with all the other (Tango) icons.

**Comment by lanwen** [comment]
Tango pics little bit ugly :)

**Comment by lanwen** [comment]
In mail list answered:
>You can add such information in the pom.xml;.

>Example: https://github.com/jenkinsci/warnings-plugin/blob/master/pom.xml#L19

```xml
 <license>
      <name>Creative Commons Attribution 3.0</name>
      <url>https://creativecommons.org/licenses/by/3.0</url>
      <comments>Go mascot and logo were designed by Renée French and are covered by the Creative Commons Attribution 3.0
        license
      </comments>
    </license>
```

How it can be added to build.gradle?

**Comment by lanwen** [comment]
Added `LICENSE.txt` file with info about picture. Think it will be enought

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Picture of the factory (job-dsl-plugin/src/main/webapp/img/factory.png) was downloaded from
```
Comment (KostyaSha) [inline LICENSE.txt]: I think It should contain plugin license as first entry and then mention about picture license

Comment (lanwen) [inline LICENSE.txt]: done

Comment (KostyaSha) [inline LICENSE.txt]: This should help license scanner better detect content of the project.

**Comment by daspilker** [comment]
+1 for the license file.

I agree with @daniel-beck that we should choose an icon from the Tango icon set if possible. Replacing a unflattering (because it's commonly used) icon which an icon from a different icon set does is not an improvement IMHO.

When searching for [generator icon](https://www.google.com/search?q=generator+icon&tbm=isch), a lot of icons showing gears turn up. So how about using one of the gears icons from the Tango icon set ([Applications-system.svg](http://commons.wikimedia.org/wiki/Tango_icons#/media/File:Applications-system.svg) or [Emblem-system.svg](http://commons.wikimedia.org/wiki/Tango_icons#/media/File:Emblem-system.svg)) to visualize generated items? Those are already included in Jenkins as gear and gear2 (at least in some sizes), but [not widely used](https://github.com/jenkinsci/jenkins/search?q=gear+OR+gear2) (in contrast to the folder icon).

**Comment by daniel-beck** [comment]
We can also use icons from the RRZE icon set: https://github.com/RRZE-PP/rrze-icon-set/tree/master/tango

core's icon for tags is from that set.

**Comment by lanwen** [comment]
The RRZE Icon Set is licenced under a [Creative Commons Licence](http://creativecommons.org/licences/by-sa/3.0/). (Same license)

**Comment by lanwen** [comment]
How about 
For dsl job view
im 

And for seed action in generated jobs?
im

Here one more (can be used in both places)
im
?

**Comment by daspilker** [comment]
I like the last one. Please update the pull request to use that icon.

**Comment by daniel-beck** [comment]
@daspilker PR has been updated.

While we're at it, the icon for generated configs doesn't really fit either.

**Comment by daspilker** [comment]
I replaced the icon for the generated configs by the directory icon.

**Pull request merged**